config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
+
+type Config struct {
+	Taskcluster struct {
+		ClientId    string
+		AccessToken string
+		Certificate string
+	}
+	ProvisionerId string
+	WorkerGroup   string
+	WorkerId      string
+	Capacity      int
+	QueueService  struct {
+		ExpirationOffset int
+	}
+}
```

Comment:
I thought we were considering to use `CompositeSchema` for config loading..... So plugins and engines could specify config to be loaded... Obviously, we wouldn't want to validate the config against their subschemas until we've loaded the config and decided which engines and plugins to load... 

Hmm... or maybe that is just a sub key of the config specified here... actually that could work too :)